mycroft/client/enclosure/__init__.py

```diff
@@ -35,6 +35,10 @@
     wait_while_speaking
 from mycroft.util.audio_test import record
 from mycroft.util.log import LOG
+if sys.version_info[0] < 3:
```

Comment:
I think we should decide to either always check `sys.version_info` or always try and except.

Reply:
Good call. Will update to use sys.version... and not try/except